### PR TITLE
Remove roles from group modal

### PR DESCRIPTION
**Pull request by PanSpagetka**
Add confirmation modal for deleting roles from group.
https://projects.engineering.redhat.com/browse/RHCLOUD-4075

Screenshot from 2020-01-27 14-40-05
Screenshot from 2020-01-27 14-39-49

### CODE REVIEW

**Comment by mmenestr** [comment]
@PanSpagetka Can you put the warning icon on the left of the header, like in the mocks? Trying to follow these PF examples https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/modal#confirm-a-destructive-action.

Also, you're missing the word "the" before permissions. So it should be "These members will lose **the** permissions....."

Otherwise, looks great!

**Comment by PanSpagetka** [comment]
@mmenestr Thanks, that's exactly kind of things I can't see. Should be fixed now :)

**Comment by mmenestr** [comment]
@PanSpagetka Icon is perfect now thank you! Just one more fix for the wording in the modals:
They should say "Members in _THE_ xxx group will lose _THE_ permissions in the xxx role."
and "Members in _THE_ xxx group will lose _THE_ permissions in these X roles"

Just missing some "the"s haha

**Comment by PanSpagetka** [comment]
@mmenestr I have added some "the"s  and "s" hopefully all English grammar intricacies are now resolved :laughing:

File: src/smart-components/group/role/group-roles.js [hunk 1]
```diff
@@ -104,35 +114,59 @@ const GroupRoles = ({
...
-            onClick: () => removeRoles(selectedRoles)
+            variant: 'danger'
+          },
+          onClick: () => {
+            setConfirmDelete(() => () => removeRoles(uuid, selectedRoles.map(role => role.uuid), () => fetchRolesForGroup(pagination)(uuid)));
+            setDeleteInfo({
+              title: 'Remove roles?',
+              confirmButtonLabel: selectedRoles.length > 1 ? 'Remove roles' : 'Remove role',
+              text: (selectedRoles.length > 1
+                ? <p>Members in the <b>{ `${name}` }</b> group will lose the permissions in these <b> { `${selectedRoles.length}` }</b> roles.</p>
```
Comment (karelhala) [inline src/smart-components/group/role/group-roles.js]: Can you make this more generic? Like a function where you can replace certain parts of text. Here is differs in `the` vs `these`  and `role` vs `roles`

**Pull request merged**